### PR TITLE
Updated Quickstart Footers

### LINKED ISSUE #4849: Update quickstart footers in getting started category
### What article on docs.github.com is affected?

- [ ] "[Set up Git](https://docs.github.com/en/github/getting-started-with-github/set-up-git)"
- [ ] "[Create a repo](https://docs.github.com/en/github/getting-started-with-github/create-a-repo)
- [ ] "[Fork a repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)"
- [ ] "[Be social](https://docs.github.com/en/github/getting-started-with-github/be-social)"

### What part(s) of the article would you like to see updated?

**Problem:** Each article in the Quickstart map topic (https://docs.github.com/en/github/getting-started-with-github/quickstart) has a footer that links to each article in the category. These footers are inconsistently formatted and their purpose is unclear. A new user may think they have to go through these articles in a prescribed order or be confused that each footer asks “What do you want to do next?” when they do not know what they want to do.
- The footer also links to https://github.community/ and implies that it’s a social hangout area of the site, this should be updated

**Solution:** 
- Create a content design plan [using this template](https://github.com/github/docs-content/blob/main/contributions/open-source-content-design-template.md) and post it in this issue.
- Update the footer to give the quickstart audience a better understanding of what to click next. 
- Remove incorrect/outdated information.

### Additional information

This issue is for a special project and is not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #4849 

### What's being changed:

Updated the footers in Quickstart map topics,
 - [Set up Git](https://docs.github.com/en/github/getting-started-with-github/quickstart/set-up-git)
 - [Create a Repo](https://docs.github.com/en/github/getting-started-with-github/quickstart/create-a-repo)
 - [Fork a Repo](https://docs.github.com/en/github/getting-started-with-github/quickstart/fork-a-repo)
 - [Be Social](https://docs.github.com/en/github/getting-started-with-github/quickstart/be-social)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
🙌 You're on a roll @codewithdev 🎉 I'll get this triaged for review ⚡

**Comment by felicitymay** [comment]
Hi @codewithdev - an internal project moved these articles to a new location, which caused these conflicts. I'll checkout your branch locally and see if I can resolve them.

**Comment by felicitymay** [comment]
Hi @codewithdev - I haven't worked with forks for a while and had forgotten that it would be slightly more complicated to get the changes from `main`. Would you be able to update the `main` branch of your fork from the `docs` repository?

You can do this in the UI here: https://github.com/codewithdev/docs

image

**Comment by codewithdev** [comment]
Sure I would,

**Comment by codewithdev** [comment]
@felicitymay Fortunately, my main branch is `even` with the upstream repository. Also, I can't update the branch for this PR as the path of the files has been changed and it has conflicts. Let me know what I can do? 🙄

**Comment by felicitymay** [comment]
@codewithdev  - thanks for updating the `main` branch for your fork 🙇🏻‍♀️ 

I've pushed a commit that merges the changes from `main` into your branch as follows:

1. Made a local clone of your repository.
2. Updated my local copy of your `main` branch with the latest changes.
3. Checked out your branch.
4. Merged `main` into your branch.
5. I was expecting to need to resolve some conflicts, but Git was able to resolve the changes from `main` without help.
6. Pushed the resulting merge commit to your branch on GitHub.

I'll take a look at your changes soon.

File: content/get-started/quickstart/fork-a-repo.md
Comment (felicitymay) [inline content/get-started/quickstart/fork-a-repo.md:157]: Updating to match the text in the article above ☝🏻 

```suggestion
Each repository in {% data variables.product.product_name %} is owned by a person or an organization. You can interact with the people, repositories, and organizations by connecting and following them on {% data variables.product.product_name %}. For more information see "[Be social](/articles/be-social)."
```

File: content/get-started/quickstart/fork-a-repo.md
Comment (felicitymay) [inline content/get-started/quickstart/fork-a-repo.md:159]: This is difficult. You've used a "reusable" which is good for making articles consistent and easier to maintain. 😄 

However, this reusable was written as a bullet point - so it starts with a capital letter which makes it unsuitable for use within a sentence. 😞 

I've checked where this is used, and it is used only in these four articles that you're editing. So here I think the best solution would be:

1. Edit the text of the reusable so that it matches what you want to say in all of these articles. It is stored here: https://github.com/github/docs/blob/main/data/reusables/support/connect-in-the-forum-bootcamp.md. I'd recommend something like this, which is based on one of the articles above:
    `{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/).`
2. Replace the current text about the support community in each of these articles with just `{% data reusables.support.connect-in-the-forum-bootcamp %}`.

**Review by felicitymay — COMMENTED** [review]
@codewithdev - thank you so much for taking the time to think about what a new user needs to know when they finish working through each of these articles. Your proposals will help them decide what to do next 💖 

I've added a lot of comments, but most of them are just for small changes. Since you're an active contributor, I've taken the time to add some explanations of most of the suggested changes. 

For smaller pull requests, we normally commit the changes immediately to speed up the turn around time for merging the work. Since this is a larger pull request and you've put a lot more thought and work into it, I'm going to leave you to take a look at the suggestions and commit them. If you have questions about any of them, let me know.

Also, let me know if you have any questions about the suggestion to update the `{% data reusables.support.connect-in-the-forum-bootcamp %}` reusable.

File: content/get-started/quickstart/set-up-git.md
Comment (codewithdev) [reply content/get-started/quickstart/set-up-git.md:61]: Yeah, Sometimes I wrote that too quickly 😄 and forgotten about the review on my own.

File: content/get-started/quickstart/fork-a-repo.md
Comment (Benzfccr7) [reply content/get-started/quickstart/fork-a-repo.md:151]: 😁

**Review by Benzfccr7 — DISMISSED** [review]

File: content/get-started/quickstart/create-a-repo.md
Comment (codewithdev) [reply content/get-started/quickstart/create-a-repo.md:84]: Thanks ✨, I will make sure to use the same name when referring to a link within the article.

File: content/get-started/quickstart/fork-a-repo.md
Comment (codewithdev) [reply content/get-started/quickstart/fork-a-repo.md:157]: I actually missed that point. However, these reusable are quite handy in terms of adding into the context.

**Comment by codewithdev** [comment]
@felicitymay I've updated the internal docs where reusable (connect-in-the-forum-bootcamp.md) is located.

 Since, this reusable is used only on these four articles thus I've applied all the changes that you've mentioned in the comment.
Also I've updated the necessary changes that you've suggested and replaced the community support with just, `{% data reusables.support.connect-in-the-forum-bootcamp %}`

Let me know if there need some certain changes in the support community reusable.

**Comment by felicitymay** [comment]
Thanks for making all those changes. 💖 

I'm not sure why the deployment to staging is still showing as pending, I'll update your branch with changes from `main` which should retrigger the deployment. 🤞🏻 

Once we can see the articles on staging, we can have a final check for anything that's slipped through the review, and then merge 😄

**Comment by codewithdev** [comment]
> Thanks for making all those changes. 💖

> Once we can see the articles on staging, we can have a final check for anything that's slipped through the review, and then merge 😄

Thats' Great. Waiting for the Final Review. 🕺

**Comment by codewithdev** [comment]
Waiting for the Final Review 👀

**Comment by felicitymay** [comment]
> There needs a little update on the reusable (`docs/data/reusables/support/connect-in-the-forum-bootcamp.md`) and we can remove that extra ". dot"

That's a great point  - well spotted ✨ 

In this case, it would be better to keep the "dot" (period) in the reusable and delete it from the articles. We try to keep reusables as full sentences that can be reused easily. For more information, see [Reusables and variables](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#reusables-and-variables).

I've spotted a couple more tiny things that I missed previously - they're all so small, it's probably quicker for me to add suggestions for them, commit them all, and then your pull request will be ready to merge.

**Comment by felicitymay** [comment]
I should have said - apart from the very small things I spotted, this is now looking great. Congratulations on making such good use of the reusable too 👍🏻

File: data/reusables/support/connect-in-the-forum-bootcamp.md
Comment (felicitymay) [inline data/reusables/support/connect-in-the-forum-bootcamp.md:1]:
```suggestion
{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/).
```

File: content/get-started/quickstart/set-up-git.md
Comment (felicitymay) [inline content/get-started/quickstart/set-up-git.md:65]:
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

File: content/get-started/quickstart/fork-a-repo.md
Comment (felicitymay) [inline content/get-started/quickstart/fork-a-repo.md:157]:
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

File: content/get-started/quickstart/create-a-repo.md
Comment (felicitymay) [inline content/get-started/quickstart/create-a-repo.md:90]:
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

File: content/get-started/quickstart/be-social.md
Comment (felicitymay) [inline content/get-started/quickstart/be-social.md:79]:
```suggestion
- {% data reusables.support.connect-in-the-forum-bootcamp %}
```

**Review by felicitymay — DISMISSED** [review]
I'm going to move the "dot" (period) back into the reusable and add a missing "the" but otherwise this looks great and will be ready to merge with these very minor adjustments.

Thanks for your patience with this process ✨

**Review by felicitymay — APPROVED** [review]

**Comment by codewithdev** [comment]
Learned many things out from this PR. Thanks, @felicitymay for the prime suggestions and reviews. ✨

**Comment by felicitymay** [comment]
> Learned many things out from this PR.

I'm glad it was useful. It's difficult for a reviewer to get the balance right. We don't want to make the process too slow so we often just make changes, but you seemed to want to know more.  ✨

**Pull request merged**
